be/src/exprs/vectorized/bitmap_functions.cpp

```diff
@@ -366,4 +367,38 @@ ColumnPtr BitmapFunctions::bitmap_min(FunctionContext* context, const starrocks:
     return builder.build(ColumnHelper::is_all_const(columns));
 }
 
+ColumnPtr BitmapFunctions::base64_to_bitmap(FunctionContext* context, const starrocks::vectorized::Columns& columns) {
+    ColumnViewer<TYPE_VARCHAR> viewer(columns[0]);
+    size_t size = columns[0]->size();
+    ColumnBuilder<TYPE_OBJECT> builder(size);
+    std::unique_ptr<char[]> p;
+
+    for (int row = 0; row < size; ++row) {
+        if (viewer.is_null(row)) {
+            builder.append_null();
+            continue;
+        }
+
+        auto src_value = viewer.value(row);
+        int ssize = src_value.size;
+        if (ssize == 0) {
+            builder.append_null();
+            continue;
+        }
+
+        p.reset(new char[ssize + 3]);
```

Comment:
Could we create a new `p`, only when the size of `row` is greater than that of `row-1`?

Reply:
i do not know which case "columns[0]->size()" will greater than 1, at present it is 1, so for loop only run 1 times

Reply:
@aaawuanjun Maybe we shouldn't assume this precondition, because we don't know how users use it. As for the extreme scenario, there maybe a column `c` storing the base64-encoded varchar. Then if we use `base64_to_bitmap(t)`, the size of columns will be greater than 1. 
And this optimization doesn't have bad effects.